docs/create-packages/Select-assemblies-referenced-by-projects.md

```diff
@@ -9,25 +9,44 @@ ms.topic: conceptual
 
 # Select Assemblies Referenced By Projects
 
-Explicit assembly references allows a subset of assemblies to be used for IntelliSense and compiling, while all assemblies are available at run-time. `PackageReference` and `packages.config` work differently, and as a result package authors need to take care to create the package to be compatible with both project types.
+Assemblies are used in two different ways during a build. The first is for compile, which allows the package consumer's code to compile against APIs in the assembly, and for Intellisense to give suggestions. The second is runtime, where the assembly is copied to the `bin` directory and is used at runtime.
```

Comment:
I feel like the setup is gone here. 

Looking at the rendered doc: https://github.com/NuGet/docs.microsoft.com-nuget/blob/a0f8607854d354a9aa7f7e9da6c8beef0225366c/docs/create-packages/Select-assemblies-referenced-by-projects.md 

It's not clear what I'm trying to achieve beyond the title